src/main/java/org/osiam/Osiam.java

```diff
@@ -2,20 +2,16 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.ryantenney.metrics.spring.config.annotation.EnableMetrics;
-import com.zaxxer.hikari.HikariConfig;
-import com.zaxxer.hikari.HikariDataSource;
+import com.zaxxer.hikari.*;
```

Comment:
I don't know why I changed the IntelliJ settings yesterday to use wildcard imports, but I'm also not a fan. I would like to introduce this as a default coding style for all osiam repositories.